test/testcafe/spec/GenericErrorHandling_spec.js

```diff
@@ -39,4 +39,5 @@ test.requestHooks(securityAccessDeniedMock)('should be able display error when r
   const terminalPage = await setup(t);
   await terminalPage.waitForErrorBox();
   await t.expect(terminalPage.getErrorMessages().getTextContent()).eql('You do not have permission to perform the requested action.');
+  await t.expect(terminalPage.getBeaconClass()).notContains('mfa-okta-email');
```

Comment:
is there a specific class that it does have?

Reply:
It has these common classes `okta-sign-in-beacon-border auth-beacon-border`, however the intent here is that the error view does not have email specific class added by the method `AuthenticatorUtil. getIconClassNameForBeacon`, hence checking for `notContains`